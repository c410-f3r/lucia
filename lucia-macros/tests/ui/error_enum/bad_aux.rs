#[lucia_macros::pkg(api(Foo), data_format(json))]
mod pkg {
  #[pkg::aux]
  impl Foo {
    type Foo = i32;
  }

  #[pkg::req_data]
  struct ReqData(
    i32
  );

  #[pkg::res_data]
  struct ResData;
}

fn main() {
}
