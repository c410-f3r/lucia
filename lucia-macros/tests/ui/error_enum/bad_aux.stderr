error: #[pkg::aux] must be an item implementation with none, one `#[pkg::aux_data]`, one `#[pkg::aux_params]` or both `#[pkg::aux_data]` and `#[pkg::aux_params]`
 --> tests/ui/error_enum/bad_aux.rs:3:10
  |
3 |   #[pkg::aux]
  |          ^^^
